 
Fails to create debug build of Qt projects on mingw
http://bugzilla.redhat.com/653674
http://bugreports.qt.nokia.com/browse/QTBUG-14467

--- qt-everywhere-opensource-src-4.7.1/mkspecs/features/qt_functions.prf.orig	2010-11-16 03:12:17.000000000 +0500
+++ qt-everywhere-opensource-src-4.7.1/mkspecs/features/qt_functions.prf	2010-11-16 03:12:50.000000000 +0500
@@ -62,7 +62,7 @@ defineTest(qtAddLibrary) {
     }
     isEmpty(LINKAGE) {
        if(!debug_and_release|build_pass):CONFIG(debug, debug|release) {
-           win32:LINKAGE = -l$${LIB_NAME}$${QT_LIBINFIX}d
+           win32:LINKAGE = -l$${LIB_NAME}d$${QT_LIBINFIX}
            mac:LINKAGE = -l$${LIB_NAME}$${QT_LIBINFIX}_debug
        }
        isEmpty(LINKAGE):LINKAGE = -l$${LIB_NAME}$${QT_LIBINFIX}
--- qt-everywhere-opensource-src-4.7.1/mkspecs/features/win32/windows.prf.orig	2010-11-16 03:11:34.000000000 +0500
+++ qt-everywhere-opensource-src-4.7.1/mkspecs/features/win32/windows.prf	2010-11-16 03:11:51.000000000 +0500
@@ -6,7 +6,7 @@ contains(TEMPLATE, ".*app"){
 
     qt:for(entryLib, $$list($$unique(QMAKE_LIBS_QT_ENTRY))) {
         isEqual(entryLib, -lqtmain): {
-            CONFIG(debug, debug|release): QMAKE_LIBS += $${entryLib}$${QT_LIBINFIX}d
+            CONFIG(debug, debug|release): QMAKE_LIBS += $${entryLib}d$${QT_LIBINFIX}
             else: QMAKE_LIBS += $${entryLib}$${QT_LIBINFIX}
         } else {
             QMAKE_LIBS += $${entryLib}
